x-pack/plugins/security_solution/public/common/components/exceptions/types.ts

```diff
@@ -11,10 +11,16 @@ import {
   Entry,
   ExceptionListItemSchema,
   CreateExceptionListItemSchema,
+  NamespaceType,
   OperatorTypeEnum,
   OperatorEnum,
 } from '../../../lists_plugin_deps';
 
+export interface ExceptionListItemIdentifiers {
```

Comment:
Note, this is specifically for the items, not the exception lists. Identifying the lists now requires a `type`